src/plugin/service/Key.h

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2015 Samsung Electronics Co.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+/**
+ * @file        Key.h
```

Comment:
It's also a good name I think.

However this file defines a Key type, so following convention used in a project I named the file after defined type.

There is another example of similar file PolicyResultSerialization.h, which does not define a type but adds a serialization functionality to the PolicyResult type defined elsewhere and it is named TYPESerialization.h

What do You think about this naming convention from this point of view ?